Engineering/IntegrationTesting

```diff
@@ -0,0 +1,43 @@
+Integration Testing
+
+Goals
+
+Integration testing (sometimes called integration and testing, abbreviated I&T) is the phase in software testing in which individual software modules are combined and tested as a group. It occurs after unit testing and before validation testing. Integration testing takes as its input modules that have been unit tested, groups them in larger aggregates, applies tests defined in an integration test plan (during ADS) to those aggregates, and delivers as its output the integrated system ready for system testing. 
+
+Evidence and Measures
+
+Integration tests should run in a different build pipeline because it takes them long time to complete, and require real resources to be used (e.g. databases). Usually they can run during the nightly builds, or manually upon request.
+
+General Guidance
+
+Integration testing does not happen at the end of the development cycle, but rather it is conducted simultaneously during the development process. So in most of the times, all the modules are not actually available to test and here is what the challenge comes to test something which does not exist!
+
+If -
```

Comment:
These are examples of integration tests, rather than an exhaustive list of what integration tests are (rephrasing)

```suggestion
If a test tests separately developed modules that interact with each other, such as:
```